apollo-router/src/warp_http_server_factory.rs

```diff
@@ -340,10 +340,11 @@ where
         })
 }
 
+// TODO: do we actually wanna track query on the info level here?
```

Comment:
Yes. We need it in the apollo telemetry. Maybe worth adding a comment here.

// graphql_request is traced at the info level so that it can be processed normally in apollo telemetry.

Reply:
sounds good, updating the comment then!